radixdlt/src/integration/java/com/radixdlt/integration/distributed/deterministic/tests/consensus/DifferentTimestampsCauseTimeoutTest.java

```diff
@@ -189,27 +107,64 @@ protected void configure() {
 			.run();
 	}
 
+	private void addTwoViews(LinkedList<Pair<ChannelId, Class<?>>> processingSequence) {
+		// Proposal here has genesis qc, which has no timestamps
+		processingSequence.add(Pair.of(ChannelId.of(1, 1), Proposal.class)); // Messages to self first
```

Comment:
I've found that often it's convenient to have static factory methods named after the class name. This enables use of static imports for such methods with no conflicts. Resulting code is slightly more concise:

`pair(channelId(1, 1), Proposal.class)`